### DISCUSSION TITLE
Autofill Date and Time for batch mode for Shadow Generator does not work

### DISCUSSION THREAD

**Discussion by UsulPaul** [UMEP-dev/UMEP#296] [Q&A]
I don't know if this is an issue, but some help would be highly appreciated. I have a raster, larger than 4000x4000 so I decided to tile it. Ended up with around 200 tiles; tried running this with the batch mode from qgis, autofill works fine on all other columns, except those related to date (DATEINI) and time (ONE_SHADOW), so to make this work, I have to Ctrl+V a lot. Is there any way around this?
![image](https://user-images.githubusercontent.com/7231322/123521802-a4644900-d6c1-11eb-8669-c4f8aec21ad7.png)

**Comment by biglimp** [comment]
At the moment, I have no suggestion. We are planning an update of this tool as from QGIS version 3.20 there is a new function called **QgsProcessingParameterDateTime** that might solve your issue.

**Reply by UsulPaul** [reply]
Got it, thank you!
